config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/PolicyEntityBuilder.java

```diff
@@ -223,6 +223,16 @@ static void prepareEncapsulatedAssertion(Policy policy, Bundle bundle, Document
     private static String findEncassReferencedGuid(Policy policy, Bundle bundle, Element encapsulatedAssertionElement, String name) {
         LOGGER.log(Level.FINE, "Looking for referenced encass: {0}", name);
         final AtomicReference<Encass> referenceEncass = new AtomicReference<>(bundle.getEncasses().get(name));
+        if (referenceEncass.get() == null) {
```

Comment:
we should consider this as missing even if it is not found in dependency bundle as well

Reply:
So this should be added after dependency bundle check

Reply:
done